media-libs/opencv/opencv-3.2.0-r1.ebuild

```diff
@@ -233,20 +269,19 @@ src_configure() {
 		-DENABLE_SOLUTION_FOLDERS=OFF
 		-DENABLE_PROFILING=OFF
 		-DENABLE_COVERAGE=OFF
-		-DENABLE_OMIT_FRAME_POINTER=OFF
+		-DENABLE_OMIT_FRAME_POINTER=${ENABLE_OMIT_FRAME_POINTER}
```

Comment:
-fomit-frame-pointer is not use flag, so I can't use it with usex.
From https://devmanual.gentoo.org/ebuild-writing/eapi/: 
>USAGE: usex <USE flag> [true output] [false output] [true suffix] [false suffix]]

I enter:

`$(usex $(get-flag -fomit-frame-pointer))`
And:
```
y-xeon ~ # emerge opencv -v
...
 * ERROR: media-libs/opencv-3.2.0-r1::gentoo failed (configure phase):
 *   USE Flag '-fomit-frame-pointer' not in IUSE for media-libs/opencv-3.2.0-r1
 * 
 * Call stack:
 *          ebuild.sh, line  115:  Called src_configure
 *        environment, line 6168:  Called usex '-fomit-frame-pointer'
 *   phase-helpers.sh, line  162:  Called use '-fomit-frame-pointer'
 *   phase-helpers.sh, line  200:  Called die
 * The specific snippet of code:
 *                              die "USE Flag '${u}' not in IUSE for ${CATEGORY}/${PF}"
```